packages/client-core/src/user/services/AuthService.ts

```diff
@@ -679,6 +679,21 @@ export const AuthService = {
     }
   },
 
+  async validateUser(email: string): Promise<boolean> {
+    try {
+      const identityProviders = await API.instance.service(identityProviderPath).find({
+        query: {
+          email: email.toLowerCase()
+        },
+        paginate: false
```

Comment:
On client side paginate: false doesnt work in params.

Why is it required all together?